.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,66 @@
+# This needs all the previous artifacts to be created:
+# the image should be published, the repo should be tagged, the helm
+# chart pushed. It's assumed everything is working, and we are doing
+# a last final test at the release creation time to _ensure_ we can
+# release. At this point it's still time to back off in case of problem,
+# not publish the release and iterate on tags.
+
+name: Smoke test
+
+on:
+  push:
```

Comment:
we might want to remove the push bit. I think it's convenient, but it might not make a whole lot of sense.